.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   test:
     working_directory: ~/likecoin-button
     docker:
-      - image: node:10
+      - image: node:14
```

Comment:
Would we want to update firebase-functions to node 14 also?

Reply:
Do you mean the `firebase-functions` [package](https://www.npmjs.com/package/firebase-functions)?

Reply:
i mean the engines in functions package json, which control the node version of deployed functions